package.json

```diff
@@ -14,6 +14,7 @@
     "fmt": "prettier --write .",
     "test": "jest",
     "typecheck": "tsc",
+    "regenerate-apis": "node -r esbuild-register scripts/commands/regenerateApiDocs.ts",
```

Comment:
Regenerate is a hard word to spell, imo. How about `regen-apis`? Matches `gen-api`.